docs/subsystems/dependencies.md

```diff
@@ -203,14 +139,7 @@ highlighting. The system is largely managed by the code in
   `ignore_missing_imports` for the new library. See
   [our mypy docs][mypy-docs] for more details.
 
-### Upgrading packages
-
-See the [README][requirements-readme] file in `requirements/` directory
-to learn how to upgrade a single Python package.
```

Comment:
Presumably we'll replace this with some links to `uv` docs with any notes on special stuff, but that can come after the main merge.